### PR TITLE
Update CONTRIBUTING

### PR DESCRIPTION
**Pull request by torn4dom4n**
Update CONTRIBUTING

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ Contributing is easy!
 
 1. [Fork this repository](https://github.com/atom/flight-manual.atom.io#fork-destination-box)
 2. Download your fork. [(Click here for help)](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced)
-3. Create a named feature branch ([we use Github flow](https://guides.github.com/introduction/flow/index.html))
+3. Create a named feature branch ([We use Github flow](https://guides.github.com/introduction/flow/index.html))
```
Comment (50Wliu) [inline CONTRIBUTING.md]: Revert this change

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -140,7 +140,7 @@ This way when someone hovers over the image, the description of the image appear
 
 All screenshots should be taken using Atom's default themes and settings.
 
-[pngcrush]: http://pmt.sourceforge.net/pngcrush/
+Using [**PNGCRUSH**](http://pmt.sourceforge.net/pngcrush) to optimize PNG files.
```
Comment (50Wliu) [inline CONTRIBUTING.md]: This change breaks all references to pngcrush and needs to be reverted.

File: CONTRIBUTING.md (hunk 1 above)
Comment (torn4dom4n) [inline CONTRIBUTING.md]: Reverted!

File: CONTRIBUTING.md (hunk 2 above)
Comment (torn4dom4n) [inline CONTRIBUTING.md]: Reverted!

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -140,7 +140,7 @@ This way when someone hovers over the image, the description of the image appear
 
 All screenshots should be taken using Atom's default themes and settings.
 
-[pngcrush]: http://pmt.sourceforge.net/pngcrush/
+[pngcrush]: http://pmt.sourceforge.net/pngcrush
```
Comment (thomasjo) [inline CONTRIBUTING.md]: I don't understand the reason behind this change — if you navigate to the proposed URL, you end up at the original URL (with the appended forward slash). The host actually issues a 301 redirect if you attempt to navigate to the proposed URL.

Please justify this change since we're now linking to an endpoint that has a 301 permanent redirect.

Comment (torn4dom4n) [inline CONTRIBUTING.md]: This is my mistake! Sorry!

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -4,10 +4,10 @@ Contributing is easy!
 
 1. [Fork this repository](https://github.com/atom/flight-manual.atom.io#fork-destination-box)
 2. Download your fork. [(Click here for help)](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced)
-3. Create a named feature branch ([we use Github flow](https://guides.github.com/introduction/flow/index.html))
+3. Create a named feature branch [(we use Github flow)](https://guides.github.com/introduction/flow/index.html)
```
Comment (50Wliu) [inline CONTRIBUTING.md]: This has the effect of making the parentheses part of the link as well.  I don't like that, and think that the one above this should also be changed so that the parentheses are excluded from the link.

Comment (torn4dom4n) [inline CONTRIBUTING.md]: I see ```[(Click here for help)](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced)``` and think they should be the same so I changed it.

Comment (lee-dohm) [inline CONTRIBUTING.md]: This should read as:

```
2. Download your fork. Refer to [the GitHub Help site](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) if you need more information.
3. Create a named feature branch. We use the standard [GitHub Flow process](https://guides.github.com/introduction/flow/index.html).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -3,11 +3,11 @@
...
 1. [Fork this repository](https://github.com/atom/flight-manual.atom.io#fork-destination-box)
-2. Download your fork. [(Click here for help)](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced)
-3. Create a named feature branch ([we use Github flow](https://guides.github.com/introduction/flow/index.html))
-4. Make a change to the content
-5. push it to a branch
-6. send us a [Pull Request](https://github.com/atom/flight-manual.atom.io/pulls).
+2. Download your fork. Refer to [the GitHub Help site](https://help.github.com/articles/fork-a-repo/#keep-your-fork-synced) if you need more information.
+3. Create a named feature branch. We use the standard [GitHub Flow process](https://guides.github.com/introduction/flow/index.html).
+4. Make a change to the Asciidoc (or whatever you want to change),
+5. Push it to a branch
```
Comment (torn4dom4n) [inline CONTRIBUTING.md]: Thanks!

**Comment by lee-dohm** [comment]
Thanks for contributing!
